cfme/infrastructure/host.py

```diff
@@ -476,11 +476,17 @@ def cluster(self):
 
 
 @attr.s
-class HostsCollection(BaseCollection):
+class HostsCollection(ComparableMixin, BaseCollection):
     """Collection object for the :py:class:`cfme.infrastructure.host.Host`."""
 
     ENTITY = Host
 
+    from cfme.common.host_views import ProviderHostsCompareView
+    from cfme.common.host_views import HostsCompareView
```

Comment:
Why are these imports down here, avoiding import errors?

Reply:
Being lazy. I'll move to top of file.